pkg/virt-manifest/defaults.go

```diff
@@ -0,0 +1,30 @@
+package virt_manifest
+
+import (
+	"kubevirt.io/kubevirt/pkg/api/v1"
+)
+
+func AddMinimalVMSpec(vm *v1.VM) {
+	// Make sure the domain name matches the VM name
+	if vm.Spec.Domain == nil {
+		vm.Spec.Domain = new(v1.DomainSpec)
+	}
+	vm.Spec.Domain.Name = vm.ObjectMeta.Name
+
+	AddMinimalDomainSpec(vm.Spec.Domain)
+}
+
+func AddMinimalDomainSpec(dom *v1.DomainSpec) {
```

Comment:
I left them separate because it makes the unit tests much more clear/concise. It's not a big deal either way if you feel strongly.